# Resources-for-Scholarly-Big-Data

Pick up scattered pearls———Tools, Resources, datasets and venues for scholarly big data

# Tools
## Bibliographical Extraction Tool  
GROBID : a machine learning library for extracting, parsing and re-structuring raw documents such as PDF into structured TEI-encoded documents with a particular focus on technical and scientific publications
http://cloud.science-miner.com/grobid/  https://github.com/kermitt2/grobid

CERMINE:Content ExtRactor and MINEr
http://cermine.ceon.pl/cermine/task.html?task=8692584047461514230

ParsCit : ParsCit is a utility for extracting citations from research papers based on Conditional Random Fields and heuristic regularization.
https://github.com/knmnyn/ParsCit/blob/master/USAGE

FreeCite: FreeCite is an open-source application that parses document citations into fielded data. You can use it as a web application or a service
http://freecite.library.brown.edu/welcome

## PDFFigures:Extracting Figures from Research Papers
Mining Figures from Research Papers
http://pdffigures2.allenai.org/

## scholar influence analysis
Computer Science Rankings:This ranking of top computer science schools is designed to identify institutions and faculty actively engaged in research across a number of areas of computer science. 
https://github.com/emeryberger/CSrankings
http://csrankings.org/#/index?all

## Text annotation
Brat : a web-based tool for text annotation

http://brat.nlplab.org/introduction.html

## Visualization
Neviewer：Analyse the evolution of research topics in a discipline based on co-word network analysis

https://link.springer.com/article/10.1007/s11192-014-1347-y

##  DatasetSearch
Making it easier to discover datasets

https://www.blog.google/products/search/making-it-easier-discover-datasets/

# Datasets and Corpus
## PubMed
PubMed comprises more than 28 million citations for biomedical literature from MEDLINE, life science journals, and online books. Citations may include links to full-text content from PubMed Central and publisher web sites.

https://www.ncbi.nlm.nih.gov/pubmed/

## Academic Vocabulary Lists
https://www.academicvocabulary.info/

## Open Academic Graph
This data set is generated by linking two large academic graphs: Microsoft Academic Graph (MAG) and AMiner
https://www.openacademic.ai/oag/

## Microsoft Academic Graph
The Microsoft Academic Graph is a heterogeneous graph containing scientific publication records, citation relationships between publications, as well as authors, institutions, journal and conference "venues," and fields of study.
https://wsdmcupchallenge.azurewebsites.net/

## Arxiv Bluk Corpus
https://arxiv.org/help/bulk_data

## ACL Anthology
A Digital Archive of Research Papers in Computational Linguistics,The ACL Anthology currently hosts over 44,000 papers on the study of computational linguistics and natural language processing
http://aclweb.org/anthology/

## AAN 
ACL Anthology Network,Here we have collected information regarding all of the papers included in the many ACL venues. From those papers, we have created several networks, including paper citation, author citation, and author collaboration. 
http://clair.eecs.umich.edu/aan/index.php
http://tangra.cs.yale.edu/newaan/

## CCL anthology
http://www.cips-cl.org/anthology

## Relational Classification Dataset
This classification dataset contains 380 scientific publications from AAN manually classified into three research areas ("Machine Translation", "Dependency Parsing" and "Summarization"). 
http://tangra.cs.yale.edu//homepage/downloads/aan_relational/

## Publication Classification Dataset
This classification dataset contains 383 scientific publications from AAN manually classified into 31 research areas using session information. The session information was compiled using the session information from ACL, COLING and EMNLP.
http://tangra.cs.yale.edu//homepage/downloads/aan_session/

## ChestX-ray8
Hospital-scale Chest X-ray Database and Benchmarks on Weakly-Supervised Classification and Localization of Common Thorax Diseases

http://openaccess.thecvf.com/content_cvpr_2017/papers/Wang_ChestX-ray8_Hospital-Scale_Chest_CVPR_2017_paper.pdf

# Venues and Challenge with datasets
## SemEval-2018 Task 7: Semantic Relation Extraction and Classification in Scientific Papers
https://competitions.codalab.org/competitions/17422#learn_the_details-overview

## SemEval 2017 Task 10: ScienceIE - Extracting Keyphrases and Relations from Scientific Publications
http://alt.qcri.org/semeval2017/task10/

## Open Academic Data Challenge 2017 (开放学术精准画像大赛)
根据学术数据挖掘系统AMiner.org和Microsoft Academic Graph提供的数据集，提取学者的个人描述信息，分析学者的研究兴趣，以及预测学者的论文引用情况
https://biendata.com/competition/scholar/

## AAAI workshop 2015,2016——Scholarly Big Data: AI Perspectives, Challenges, and Ideas Workshop co-located with 
http://people.cs.ksu.edu/~ccaragea/aaai2016ws/program.html
https://www.aaai.org/Workshops/ws15workshops.php

## SAVE-SD 2016-2018:WWW Workshop on Semantics, Analytics and Visualisation: Enhancing Scholarly Data 
SAVE-SD 2016 http://cs.unibo.it/save-sd/2016/index.html
SAVE-SD 2017 http://oro.open.ac.uk/53280/
SAVE-SD 2018 https://save-sd.github.io/2018/index.html

## WSDM 2017 Workshop on Scholarly Web Mining (SWM 2017) 

https://ornlcda.github.io/SWM2017/program.html

## International Workshop on Mining Scientific Publications 2012-2018
https://wosp.core.ac.uk/lrec2018/ 

 https://wosp.core.ac.uk/jcdl2017/
https://wosp.core.ac.uk/jcdl2016/
https://wosp.core.ac.uk/jcdl2015/
http://project.core.ac.uk/dl2014/
http://project.core.ac.uk/jcdl2013/
http://project.core.ac.uk/jcdl2012/

## BIG 2016 CUP:(WWW 2016)
Microsoft provides the latest Microsoft Academic Search data set and an online graph query interface for BIG 2016 CUP

http://big2016.org/big-2016-cup/

## KDD Cup 2016- Whose papers are accepted the most: towards measuring the impact of research institutions
Given any upcoming top conferences such as KDD, SIGIR, and ICML in 2016, rank the importance of institutions based on predicting how many of their papers will be accepted.

https://www.kdd.org/kdd-cup/view/kdd-cup-2016/Tasks

## KDD Cup 2013- Author-Paper Identification Challenge 
Determine whether an author has written a given paper

https://www.kaggle.com/c/kdd-cup-2013-author-paper-identification-challenge

## KDD Cup 2008- Detection of Breast Cancer
early detection of breast cancer from X-ray images of the breast

https://www.kdd.org/kdd-cup/view/kdd-cup-2008

## KDD Cup 2006-Pulmonary Embolisms Detection from Image Data
identifying pulmonary embolisms from three-dimensional computed tomography data

https://www.kdd.org/kdd-cup/view/kdd-cup-2006

## KDD Cup 2004- Particle Physics and Protein Homology Prediction Task
The goal in Particle Physics task is to learn a classification rule that differentiates between two types of particles generated in high energy collider experiments. The goal of Protein Homology Prediction is to predict which proteins are homologous to a native sequence
https://www.kdd.org/kdd-cup/view/kdd-cup-2004/Tasks

## KDD Cup 2003- Network mining and usage log analysis: Citation PredictionCitation Prediction and Download Estimation
The goal of  Citation Prediction is to predict changes in the number of citations to individual papers over time.The goal of Download Estimation task is to estimate the number of downloads that a paper receives in its first two months in the arXiv.
https://www.kdd.org/kdd-cup/view/kdd-cup-2003/Tasks

## KDD Cup 2002- BioMed document; plus gene role classification
This year the competition included two tasks that involved data mining in molecular biology domains. Task 1: Information Extraction from Biomedical Articles；Task 2: Yeast Gene Regulation Prediction. The first task focused on constructing models that can assist genome annotators by automatically extracting information from scientific articles. The second task focused on learning models that characterize the behavior of individual genes in a hidden experimental setting. Both are described in more detail on the Tasks page.
https://www.biostat.wisc.edu/~craven/kddcup/tasks.html

## KDD Cup 2001- Mining Biological Databases 
KDD Cup 2001 was focused on data from genomics and drug design. Sufficient (yet concise) information was provided so that detailed domain knowledge was not a requirement for entry. A total of 136 groups participated to produce a total of 200 submitted predictions over the 3 tasks: 114 for Thrombin, 41 for Function, and 45 for Localization.

http://pages.cs.wisc.edu/~dpage/kddcup2001/
